### ISSUE TITLE
ucs - DEBUG - Unknown property lddp_transmit

### ISSUE THREAD

**Issue by albertdamien** [vesposito/easyucs#18]
Hello,

while playing with easyucs on the UCSPE emulator, I was trying to push a configuration and I got the following error:

`2021-03-16 03:13:30,980 :: INFO :: 192.168.56.132 :: lan.py :: in push_object :: Pushing UDLD Link Policy configuration: default
2021-03-16 03:13:31,041 :: INFO :: 192.168.56.132 :: lan.py :: in push_object :: Pushing Link Profile configuration: default
2021-03-16 03:13:31,112 :: INFO :: 192.168.56.132 :: lan.py :: in push_object :: Pushing Appliance Network Control Policy configuration: default
2021-03-16 03:13:31,113 - ucs - DEBUG - Unknown property lddp_transmit
Traceback (most recent call last):
  File "easyucs.py", line 813, in <module>
    main()
  File "easyucs.py", line 806, in main
    init_process(ucs_device, args, json_string)
  File "easyucs.py", line 268, in init_process
    ucs_device.config_manager.push_config(bypass_version_checks=bypass_version_checks)
  File "/home/dalbert/easyucs/config/ucs/manager.py", line 795, in push_config
    appliance_network_control_policy.push_object()
  File "/home/dalbert/easyucs/config/ucs/ucsm/lan.py", line 2238, in push_object
    mo_nwctrl_definition = NwctrlDefinition(parent_mo_or_dn=parent_mo, cdp=self.cdp_admin_state,
  File "/home/dalbert/.local/lib/python3.8/site-packages/ucsmsdk/mometa/nwctrl/NwctrlDefinition.py", line 91, in __init__
    ManagedObject.__init__(self, "NwctrlDefinition", parent_mo_or_dn, **kwargs)
  File "/home/dalbert/.local/lib/python3.8/site-packages/ucsmsdk/ucsmo.py", line 82, in __init__
    self._set_mo_prop_value(kwargs)
  File "/home/dalbert/.local/lib/python3.8/site-packages/ucsmsdk/ucsmo.py", line 106, in _set_mo_prop_value
    self.__set_prop(prop_name, prop_value)
  File "/home/dalbert/.local/lib/python3.8/site-packages/ucsmsdk/ucsmo.py", line 215, in __set_prop
    prop_meta = self.prop_meta[name]
KeyError: 'lddp_transmit'
`
  I think it's a typo in the config/ucs/ucsm/lan.py file at line 2241:

def push_object(self, commit=True):
   2231         if commit:
   2232             self.logger(message="Pushing " + self._CONFIG_NAME + " configuration: " + str(self.name))
   2233         else:
   2234             self.logger(message="Adding to the handle " + self._CONFIG_NAME + " configuration: " + str(self.name) +
   2235                                 ", waiting for a commit")
   2236 
   2237         parent_mo = "fabric/eth-estc"
   2238         mo_nwctrl_definition = NwctrlDefinition(parent_mo_or_dn=parent_mo, cdp=self.cdp_admin_state,
   2239                                                 lldp_receive=self.lldp_receive, name=self.name,
   2240                                                 uplink_fail_action=self.action_on_uplink_fail,
   **2241                                                 lddp_transmit=self.lldp_transmit, descr=self.descr,**
   2242                                                 mac_register_mode=self.mac_register_mode)
   2243         DpsecMac(parent_mo_or_dn=mo_nwctrl_definition, descr=self.descr, name="", forge=self.mac_security_forge)
   2244 
   2245         self._handle.add_mo(mo=mo_nwctrl_definition, modify_present=True)
   2246         if commit:
   2247             if self.commit(detail=self.name) != True:
   2248                 return False
   2249         return True

Thanks

**Comment by albertdamien** [comment]
Problem solved by applying this patch :

```albert@ubuntu:~/easyucs$ git diff
diff --git a/config/ucs/ucsm/lan.py b/config/ucs/ucsm/lan.py
index 87e260b..6e1fce5 100644
--- a/config/ucs/ucsm/lan.py
+++ b/config/ucs/ucsm/lan.py
@@ -2238,7 +2238,7 @@ class UcsSystemApplianceNetworkControlPolicy(UcsSystemConfigObject):
         mo_nwctrl_definition = NwctrlDefinition(parent_mo_or_dn=parent_mo, cdp=self.cdp_admin_state,
                                                 lldp_receive=self.lldp_receive, name=self.name,
                                                 uplink_fail_action=self.action_on_uplink_fail,
-                                                lddp_transmit=self.lldp_transmit, descr=self.descr,
+                                                lldp_transmit=self.lldp_transmit, descr=self.descr,
                                                 mac_register_mode=self.mac_register_mode)
         DpsecMac(parent_mo_or_dn=mo_nwctrl_definition, descr=self.descr, name="", forge=self.mac_security_forge)
 
```

**Comment by vesposito** [comment]
Hello,

I actually have already fixed this internally a few days ago, but haven't had time to push the update yet.

I might push a new update with a few small fixes while waiting for the next SDK (ucsmsdk, ucscsdk, imcsdk) updates.

Thanks!
  Vincent

**Comment by vesposito** [comment]
Fixed in v0.9.8.5
